File: 2020_November/25.cpp
class Solution {
public:
    int smallestRepunitDivByK(int K) {
        if (K % 2 == 0 || K % 5 == 0) return -1;
        int g = 0;
        for (int i = 0; i < K; i++){
            g = (g*10+1) % K;
            if (g == 0) return i+1;
        }
        return -1;
    }
};

File: 2021_Aug/25.py
class Solution:
    def judgeSquareSum(self, c: int) -> bool:
        for i in range(0,int(math.sqrt(c//2)) + 2):
            print(i)
            if c - i*i >= 0 and int(sqrt((c - i*i)))**2 == c - i*i:
                return True
        return False